internal/server/handlers.go

```diff
@@ -635,11 +635,10 @@ func (a *API) UpdateIdentityInfoFromProvider(c *gin.Context) error {
 	return access.UpdateIdentityInfoFromProvider(c, oidc)
 }
 
-func (a *API) providerClient(c *gin.Context, provider *models.Provider, redirectURL string) (providers.OIDCClient, error) {
-	if val, ok := c.Get("oidc"); ok {
+func (a *API) providerClient(ctx context.Context, provider *models.Provider, redirectURL string) (providers.OIDCClient, error) {
```

Comment:
`gin.Context` implements `context.Context`, so none of the callers needed to change here.